src/logic/hooks/__tests__/useAsync.test.ts

```diff
@@ -9,7 +9,7 @@ describe('useAsync tests', () => {
 
     await waitForNextUpdate()
 
-    expect(result.current).toEqual({ result: 'success', error: undefined })
+    expect(result.current).toEqual({ result: 'success', error: undefined, isLoading: false })
```

Comment:
I would add an extra here test to ensure `isLoading` functions correctly.

Reply:
I am already testing it in the last tests:

![Captura de pantalla 2022-04-11 a las 18 11 14](https://user-images.githubusercontent.com/26763673/162784331-59bd68a5-58d2-41eb-98ae-80b9119b1e73.png)
![Captura de pantalla 2022-04-11 a las 18 11 51](https://user-images.githubusercontent.com/26763673/162784347-0b0de208-c893-40fa-b993-fc7c4ad7d99a.png)

But I can create a complete new one if you want.